QualityControl/test/public/features/runMode.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @stylistic/js/max-len */
```

Comment:
Nothing should be above the licence header. 
If you really need to disable an eslint rule, please do so after the header